editor/editor_node.cpp

```diff
@@ -584,7 +586,6 @@ void EditorNode::_notification(int p_what) {
 
 			if (unsaved_cache != (saved_version != editor_data.get_undo_redo().get_version())) {
```

Comment:
The `if` can be removed now as this is equivalent to `if (foo = false) { foo = true; }` which is the same as just always doing `foo = true;`.